x/params/types/table.go

```diff
@@ -72,7 +74,7 @@ func (t KeyTable) RegisterParamSet(ps ParamSet) KeyTable {
 }
 
 func (t KeyTable) maxKeyLength() (res int) {
-	for k := range t.m {
+	for _, k := range maps.Keys(t.m) {
 		l := len(k)
 		if l > res {
```

Comment:
This code isn't sorting the keys, so AFAICT doesn't produce behavior any different than ranging over the map directly?